charts/nginz/values.yaml

```diff
@@ -444,6 +444,11 @@ nginx_conf:
       - all
       doc: true
       oauth_scope: conversations_code
+    - path: /conversations/join
```

Comment:
Do I understand correctly that this is an authenticated-endpoint-only?

I don't have enough context here to actually approve this PR. The rate limit bit is fine; but this adds a new public endpoint which I don't know exactly how it works what it does and how product wise it's supposed to be used.

But the existing guest links use `/conversations/code` (not /join). So shouldn't this modify the existing API's rate limiting and not introduce this new endpoint? Do client teams switch over to this other endpoint? Where was this implemented to start with?

Reply:
> Do I understand correctly that this is an authenticated-endpoint-only?
> 
> I don't have enough context here to actually approve this PR. The rate limit bit is fine; but this adds a new public endpoint which I don't know exactly how it works what it does and how product wise it's supposed to be used.
> 
> But the existing guest links use `/conversations/code` (not /join). So shouldn't this modify the existing API's rate limiting and not introduce this new endpoint? Do client teams switch over to this other endpoint? Where was this implemented to start with?

This PR does not introduce a new endpoint. It was already there before and did match the following directive:

```yaml
    - path: /conversations
      envs:
      - all
```

I just created a new entry, so that the configuration is only applied to the path `/conversations/join`.

Reply:
Ah I see. Okay. That's fine. Thanks for the context.